public/app/plugins/datasource/grafana-azure-monitor-datasource/components/ResourcePicker/ResourcePicker.tsx

```diff
@@ -170,13 +153,53 @@ const ResourcePicker = ({
                 />
               </>
             )}
-
+            <Collapse
+              collapsible
+              label="Advanced"
+              isOpen={isAdvancedOpen}
+              onToggle={() => setIsAdvancedOpen(!isAdvancedOpen)}
+            >
+              <Label htmlFor={`input-${internalSelectedURI}`}>
+                <h6>
+                  Resource URI{' '}
+                  <Tooltip
+                    content={
+                      <>
+                        <p>
+                          You can manually edit the{' '}
+                          <a
+                            href="https://docs.microsoft.com/en-us/azure/azure-monitor/logs/log-standard-columns#_resourceid"
+                            rel="noopener noreferrer"
+                            target="_blank"
+                          >
+                            resource uri{' '}
+                          </a>
+                          or use template variables:
+                        </p>
+                      </>
+                    }
+                    placement="right"
+                    interactive={true}
+                  >
+                    <Icon name="info-circle" />
+                  </Tooltip>
+                </h6>
+              </Label>
+              <Input
+                id={`input-${internalSelectedURI}`}
+                value={internalSelectedURI}
+                onChange={(event) => setInternalSelectedURI(event.currentTarget.value)}
+                placeholder="ex: /subscriptions/$subId"
```

Comment:
Hmm. We can? My thinking is though that if you're on the edit panel page and you open the resource picker your intention is to edit the panel. Seems odd to then make someone have to click again to edit. If you do mistype something in there you can reselect things in the resource picker and it will overwrite whatever you have in there or you can click cancel?

Reply:
yeah, I was thinking on users clicking on "Advanced" out of curiosity and then editing by mistake the field but it's not something super worrying, up to you of whatever you decide.

Reply:
Since nothing updates until you click apply I think it's already safeguarded, but if folks make bugs saying they are confused maybe we can reopen!